eng/Versions.props

```diff
@@ -58,26 +52,26 @@
   -->
   <PropertyGroup Label="Automated">
     <!-- dotnet/arcade references -->
-    <MicrosoftDotNetBuildTasksArchivesVersion>8.0.0-beta.23463.1</MicrosoftDotNetBuildTasksArchivesVersion>
-    <MicrosoftDotNetCodeAnalysisVersion>8.0.0-beta.23463.1</MicrosoftDotNetCodeAnalysisVersion>
-    <MicrosoftDotNetXUnitExtensionsVersion>8.0.0-beta.23463.1</MicrosoftDotNetXUnitExtensionsVersion>
+    <MicrosoftDotNetBuildTasksArchivesVersion>9.0.0-beta.23510.4</MicrosoftDotNetBuildTasksArchivesVersion>
```

Comment:
Looks like this merged cleanly so it's the wrong version. I will fix it

Reply:
Maybe we should update arcade in the main branch to pull from the .NET 8 Eng channel first and then merge. This channel didn't exist a few days ago.